### ISSUE TITLE
Incorrect description for Quartz Cron Expression: 0 0 11 L-2 * ?

### ISSUE THREAD

**Issue by LennyKarpel** [jmrozanec/cron-utils#308]
Currently the description returned is 'at 11:00 last day of month'. Should be described as happening at 11:00 **_two days before_** the end of the month.

**Comment by jmrozanec** [comment]
@LennyKarpel Thank you for reporting this! May we ask for a PR with a test for this issue? Happy New Year!

**Comment by LennyKarpel** [comment]
@jmrozanec Happy New Year to you too !!

I am new this this. How do I create a PR if I cannot create a branch to put the changes on ?

The test would be something like:
```
@Test
public void testIssue308()
{
    CronDefinition    cronDefinition = CronDefinitionBuilder.instanceDefinitionFor( CronType.QUARTZ );
    CronParser        parser = new CronParser( cronDefinition );
    Cron              quartzCron = parser.parse( "0 0 11 L-2 * ?" );
    CronDescriptor    descriptor = CronDescriptor.instance( Locale.getDefault() );
    String            description = descriptor.describe( quartzCron );

    // not sure what the exact string 'should' be ..

    assertNotEquals( "at 11:00 two days before the last day of month", description);
}
```
anyway .. hope that helps. Let me know if I can do more.

**Comment by Naxos84** [comment]
Fork this repo.
Then add your branch (if you like). Push the changes to your forked repo.
After that you can create the pull request.

Am 30.12.2017 18:09 schrieb "Lenny Karpel" <notifications@github.com>:

**Comment by pangyikhei** [comment]
@jmrozanec I can tackle this one.

**Comment by jmrozanec** [comment]
@LennyKarpel @Naxos84 Fixed by @pangyikhei in #458!
